dotcom-rendering/src/components/Dateline.tsx

```diff
@@ -25,10 +26,10 @@ const hoverUnderline = css`
 `;
 
 // for liveblog smaller breakpoints article meta is located in the same
-// container as standfirst and needs the same styling as standfirst
-const standfirstColouring = css`
+// container as standfirst and needs the same styling as standfirst on web
+const liveBlogStyles = css`
 	${until.desktop} {
-		color: ${palette('--standfirst-text')};
+		color: var(--mobile-color);
 	}
 `;
```

Comment:
I do not think this has strictly to do with live blogs?

If we are systematically applying these styles, we could fold them into `dateLineStyles`.

Can be adressed in a follow-up.